### ISSUE TITLE
Apollo 8 interior

### ISSUE THREAD

**Issue by micahnyc** [OpenSpace/OpenSpace#2385]
Now that we have better model support, we could re-implement the apollo 8 interior capsule with just one file from here: 

https://3d.si.edu/object/3d/command-module-apollo-11:d8c63e8a-4ebc-11ea-b77f-2e728ce88125

https://legacy.3d.si.edu/apollo11cm/boxes/play-cm-2016-09-26/cm-interior-vr.html

After it's added, we should create a 2nd action for doing an earthrise moment from inside the capsule

**Comment by Belize12** [comment]
I ran across this Apollo Github while searching for Apollo APL Fintech.com, I'm not a developer so its hard for me to understand most of this stuff, am I at the right Github or is this something completely different, sorry for the question, thanks in advance.

**Comment by alexanderbock** [comment]
Is this the one that @jockekilby used?

**Comment by jockekilby** [comment]
Yeah that's the one we're using, the high-poly model. We've made some minor adjustments & added a hatch to it.
I wouldn't recommend using the high-poly model as it make OS drop to about 5fps 🐢
